snippets/UUIDGenerator.md

```diff
@@ -4,13 +4,26 @@ Generates a UUID.
 
 Use `crypto` API to generate a UUID, compliant with [RFC4122](https://www.ietf.org/rfc/rfc4122.txt) version 4.
 
+Please note that this version will only work in **Node.JS** environments!
+
 ```js
+const crypto = require("crypto");
+
 const UUIDGenerator = () =>
   ([1e7] + -1e3 + -4e3 + -8e3 + -1e11).replace(/[018]/g, c =>
-    (c ^ (crypto.getRandomValues(new Uint8Array(1))[0] & (15 >> (c / 4)))).toString(16)
+    (c ^ (crypto.randomBytes(1)[0] & (15 >> (c / 4)))).toString(16)
   );
 ```
 
+If you'd like a dependency-less UUID Generator that will work in both browsers and Node.JS, you should choose this one instead.
+
+```js
+const UUIDGenerator2 = a =>
+	a ? ((a ^ Math.random() * 16) >> a / 4).toString(16) :
```

Comment:
Math.random is not cryptographically safe, and thus not suitable for use in generating UUIDs

Reply:
Yup, `Math.random` is a bad bad idea.